src/index.tsx

```diff
@@ -3,12 +3,14 @@ import * as ReactDOM from "react-dom";
 import { Router, Route, browserHistory, hashHistory } from "react-router";
 import ContextProvider from "./components/ContextProvider";
 import App from "./components/App";
+import LogInForm from "./components/reusables/LogInForm";
 
 interface ConfigurationSettings {
   /** A token generated by the server to prevent Cross-Site Request Forgery.
       The token should be included in an 'X-CSRF-Token' header in any non-GET
       requests. */
-  csrfToken: string;
+  csrfToken?: string;
```

Comment:
You're still not using the `csrfToken`?

Reply:
It's still commented out in the `ContextProvider` and I thought it was going to be used for logging in?